common/logging/src/lib.rs

```diff
@@ -97,6 +95,21 @@ impl<'a> Write for AlignedRecordDecorator<'a> {
             self.wrapped.write(buf)
         }
     }
+}
+
+impl<'a> Write for AlignedRecordDecorator<'a> {
+    fn write(&mut self, buf: &[u8]) -> Result<usize> {
+        if buf.iter().any(|c| is_ascii_control(c)) {
+            let filtered = buf
+                .iter()
+                .cloned()
+                .map(|c| if !is_ascii_control(&c) { c } else { b'_' })
```

Comment:
I recall you changing `_` to something else. Did that not eventuate?

Reply:
I'm going to resolve this. @macladson reminded me that we only have access to one byte so we can't do funky unicode chars. `_` seems as good as anything else to me.